libretro_core_options.h

```diff
@@ -90,7 +90,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(internal_resolution),
       "Internal GPU Resolution",
       NULL,
-      "Set internal resolution multiplier. Resolutions higher than '1x (Native)' improve fidelity of 3D models at the expense of increased performance requirements. 2D elements are generally unaffected by this setting.",
+      "Choose internal resolution multiplier. Resolutions higher than '1x (Native)' improve fidelity of 3D models at the expense of increased performance requirements. 2D elements are generally unaffected by this setting.",
```

Comment:
Articles are technically not required in those cases. 
But let's say, we add one, which would we even use here, "the" or "a"? And why?

Reply:
I think it would be a "the" in this case, as an option has to be chosen, even 1x, which is the default setting that does no changes from the original resolution. But this might be a case of me thinking a bit in Spanish, rather than in English. If it's not required, you can discard this comment.

Reply:
Okay, I'll go over them again and try to add articles where it fits. Thank you for your input.